solutions/kafka-schema-registry/README.md

```diff
@@ -0,0 +1,60 @@
+The project demonstrates how to read send/receive messages serialized/deserialized with Avro to/from Kafka
+
+### Requirements
+Installed JDK 1.8+
```

Comment:
These should be bullet points or separate paragraphs (otherwise they appear on one line).  Same with `Preparation` below.